packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -397,7 +424,11 @@ class DropdownCore extends React.Component<Props, State> {
     // Figure out focus states for the dropdown after it has changed from open
     // to closed or vice versa
     initialFocusItem() {
-        const {open} = this.props;
+        const {autoFocus, open} = this.props;
+
+        if (!autoFocus) {
+            return;
+        }
```

Comment:
**suggestion (non-blocking):** Now that we aren't always focusing an item, this should maybe be renamed to `maybeFocusInitialItem()`.